openmdao/matrices/csc_matrix.py

```diff
@@ -40,7 +41,7 @@ def _post_update(self):
         """
         coo = self._coo
         # this will add any repeated entries together
-        # NOTE: this form of the ctor was used instead of self._coo.tocsc() because
+        # NOTE: this form of the vector was used instead of self._coo.tocsc() because
```

Comment:
change to:
NOTE: The CSC matrix was created in the following way instead of using self._coo.tocsc() because on older versions of scipy, self._coo.tocsc() reuses the row/col arrays and the result is ...